Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/OpenShiftRequestController.cs

```diff
@@ -415,158 +408,159 @@ internal async Task ProcessOpenShiftsRequests(string isRequestFromLogicApp)
+                            {
+                                entityToUpdate.KronosStatus = item.StatusName;
+                                entityToUpdate.ShiftsStatus = item.StatusName;
 
-                                    // Commit the change to the database.
-                                    await this.openShiftRequestMappingEntityProvider.SaveOrUpdateOpenShiftRequestMappingEntityAsync(entityToUpdate).ConfigureAwait(false);
+                                // Commit the change to the database.
+                                await this.openShiftRequestMappingEntityProvider.SaveOrUpdateOpenShiftRequestMappingEntityAsync(entityToUpdate).ConfigureAwait(false);
 
-                                    var declineMessageModel = new OpenShiftRequestApproval
-                                    {
-                                        Message = Resource.OpenShiftRequestDeclinedMessage,
-                                    };
+                                var declineMessageModel = new OpenShiftRequestApproval
+                                {
+                                    Message = Resource.OpenShiftRequestDeclinedMessage,
+                                };
 
-                                    var declineMessageModelStr = JsonConvert.SerializeObject(declineMessageModel);
-                                    var declineHttpClient = this.httpClientFactory.CreateClient("ShiftsAPI");
-                                    declineHttpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", allRequiredConfigurations.ShiftsAccessToken);
+                                var declineMessageModelStr = JsonConvert.SerializeObject(declineMessageModel);
+                                var declineHttpClient = this.httpClientFactory.CreateClient("ShiftsAPI");
+                                declineHttpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", allRequiredConfigurations.ShiftsAccessToken);
 
-                                    // Send Passthrough header to indicate the sender of request in outbound call.
-                                    declineHttpClient.DefaultRequestHeaders.Add("X-MS-WFMPassthrough", allRequiredConfigurations.WFIId);
-                                    using (var declineRequestMessage = new HttpRequestMessage(HttpMethod.Post, "teams/" + user.ShiftTeamId + "/schedule/openshiftchangerequests/" + entityToUpdate.RowKey + "/decline")
+                                // Send Passthrough header to indicate the sender of request in outbound call.
+                                declineHttpClient.DefaultRequestHeaders.Add("X-MS-WFMPassthrough", allRequiredConfigurations.WFIId);
+                                using (var declineRequestMessage = new HttpRequestMessage(HttpMethod.Post, "teams/" + user.ShiftTeamId + "/schedule/openshiftchangerequests/" + entityToUpdate.RowKey + "/decline")
```

Comment:
same here, template string would be nicer

Reply:
Done